src/Magnum/SceneGraph/OctreeDrawableGroup.h

```diff
@@ -0,0 +1,120 @@
+#ifndef Magnum_SceneGraph_OctreeDrawableGroup_h
+#define Magnum_SceneGraph_OctreeDrawableGroup_h
+/*
+    This file is part of Magnum.
+
+    Copyright © 2010, 2011, 2012, 2013, 2014, 2015
+              Vladimír Vondruš <mosra@centrum.cz>
+    Copyright © 2015 Andrea Capobianco <andrea.c.2205@gmail.com>
+    Copyright © 2015 Jonathan Hale <squareys@googlemail.com>
+
+    Permission is hereby granted, free of charge, to any person obtaining a
+    copy of this software and associated documentation files (the "Software"),
+    to deal in the Software without restriction, including without limitation
+    the rights to use, copy, modify, merge, publish, distribute, sublicense,
+    and/or sell copies of the Software, and to permit persons to whom the
+    Software is furnished to do so, subject to the following conditions:
+
+    The above copyright notice and this permission notice shall be included
+    in all copies or substantial portions of the Software.
+
+    THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+    IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+    FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL
+    THE AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+    LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING
+    FROM, OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER
+    DEALINGS IN THE SOFTWARE.
+*/
+
+/** @file
+ * @brief Class @ref Magnum::SceneGraph::Drawable, @ref Magnum::SceneGraph::DrawableGroup, alias @ref Magnum::SceneGraph::BasicDrawable2D, @ref Magnum::SceneGraph::BasicDrawable3D, @ref Magnum::SceneGraph::BasicDrawableGroup2D, @ref Magnum::SceneGraph::BasicDrawableGroup3D, typedef @ref Magnum::SceneGraph::Drawable2D, @ref Magnum::SceneGraph::Drawable3D, @ref Magnum::SceneGraph::DrawableGroup2D, @ref Magnum::SceneGraph::DrawableGroup3D
+ */
+
+#include <functional>
+
+#include "Magnum/SceneGraph/Drawable.h"
+#include "Magnum/Shapes/AbstractShape.h"
+#include "Magnum/Octree/Octree.h"
+
+namespace Magnum { namespace SceneGraph {
+
+/**
+@brief OctreeDrawableGroup
+
+A @ref DrawableGroup which makes use of view frustum culling to avoid drawing meshes
+outside of the cameras view using an @ref Magnum::Octree::Octree.
+
+This only works for three dimensional scenes.
+
+@todo doc
+
+@see @ref scenegraph, @ref BasicDrawable2D, @ref BasicDrawable3D,
+    @ref Drawable2D, @ref Drawable3D, @ref DrawableGroup
+*/
+template<class T> class OctreeDrawableGroup: public DrawableGroup<3, T> {
+    public:
+        /**
+         * @brief Constructor
+         *
+         * Adds @ref Drawable s from drawables if specified.
+         * Otherwise you can use @ref OctreeDrawableGroup::add().
+         */
+        explicit OctreeDrawableGroup():
+            DrawableGroup<3, T>(),
+            _octree(nullptr),
+            _drawables(),
+            _boundingBoxes()
```

Comment:
No need to call implicit constructors explicitly. For the others I'd prefer to do in-class initialization.